argo-workflows/install-using-helm/ui.md

```diff
@@ -44,6 +44,7 @@ metadata:
   generateName: hello-world-
   namespace: argo
 spec:
+  serviceAccountName: argo-argo-workflows-workflow-controller
```

Comment:
Thanks for your contribution. Sorry for the delay. I must have missed a notification or something.

Unfortunately, this is definitely the wrong serviceaccount to use on any workflow. A workflow is not the workflow controller. This is excessively permissive and should not be encouraged for new users.

The helm chart optionally creates a workflow-specific serviceaccount
https://github.com/argoproj/argo-helm/blob/main/charts/argo-workflows/templates/controller/workflow-sa.yaml

https://github.com/argoproj/argo-helm/blob/main/charts/argo-workflows/values.yaml#L59

which has better permissions (albeit still quite overly-permissive). This is what should be used by the workflow here.


So the install command in install.md needs to include `--set workflow.serviceAccount.create=true`
And then this line here can be changed to `serviceAccountName: argo-workflow`